app/models/auth-tokens/app-tokens-scopes.ts

```diff
@@ -91,6 +91,7 @@ export const appTokensScopes = [
       'GET /vN/timelines-rss/:username',
       'GET /vN/posts/:postId',
       'GET /vN/posts/:postId/translated-body',
+      'GET /vN/posts/:postId/referring-posts',
```

Comment:
I think I'd just call it `.../backlinks`, what do you think?

Reply:
Well, here we return not links, but posts with them :) I think, this is more accurate.

Reply:
Ok, renamed this API method to /backlinks